src/libraries/Microsoft.PowerFx.Interpreter/Marshal/TypeCoercionProvider.cs

```diff
@@ -19,18 +19,9 @@ namespace Microsoft.PowerFx
     /// </summary>
     public static class TypeCoercionProvider
     {
-        internal static FormattingInfo CreateFormattingInfo() => new FormattingInfo()
-        {
-            // $$$ can't use current culture
-            CultureInfo = CultureInfo.CurrentCulture,            
-            TimeZoneInfo = TimeZoneInfo.Local

```

Comment:
oh, I see - it came from here....  we shouldn't have had it here either.

Reply:
update to use invariantculture and move to helper.